app/server/app/routes/api.js

```diff
@@ -36,10 +42,13 @@ router.get("/epa-data", (req, res) => {
 router.get("/sam-data", (req, res) => {
   getSamData(req.user.mail)
     .then((samUserData) => {
+      // TODO: set `helpdeskUser` to true if they're a member of the correct EPA WAA groups
+      const helpdeskUser = false;
```

Comment:
@dpgalloway here you'll want to do that same check to determine if the user is a help desk member. If they are, set the `helpdeskUser` variable to true, else set it false, as it's used in the if statement below to allow helpdesk users without any SAM.gov data to still access the app.

I'm realizing now I should probably limit a help desk user's access to create new forms if they don't have any SAM.gov data, but I can do that as a separate PR.